tests/Data/Food/RecipeTest.elm

```diff
@@ -256,69 +264,77 @@ suite =
                             |> Expect.equal royalPizzaWithNoPackaging
                             |> asTest "should give the same mass including packaging or not"
                         ]
-                    , let
-                        mango =
-                            { id = Ingredient.idFromString "mango-non-eu"
-                            , mass = Mass.grams 120
-                            , country = Nothing
-                            , planeTransport = Ingredient.ByPlane
-                            }
+                    , describe "computeIngredientTransport"
+                        (case ( Ingredient.idFromString "9cbc31e9-80a4-4b87-ac4b-ddc051c47f69", Ingredient.idFromString "db0e5f44-34b4-4160-b003-77c828d75e60" ) of
```

Comment:
Nit: maybe spread this over multiple lines for legibility

Also it feels like you could try getting valid ids for egg, wheat and mango in one go at the beginning of the suite.